### ISSUE TITLE
Bypass posting restrictions when Duplicate

### ISSUE THREAD

**Issue by mihaijoldis** [restrictcontentpro/rcp-wp-job-manager#22]
If you have 3 jobs posted and your RCP membership limit is set to 3 for example you can easily bypass that by using the "duplicate" action on the job dashboard page

**Comment by mihaijoldis** [comment]
Just had another report of this. Someone who purchased today to use it and discovered this

**Comment by nosegraze** [comment]
PR: #25 

This is kind of a gross way of doing it. I'm hooking into `job_manager_user_can_edit_job` and returning `false` if the user is currently trying to duplicate and is at their limit. This hook is, frankly, irrelevant to what we're actually checking, but there don't appear to be any better hooks/filters in place. There is no `can_duplicate` type filter, and when duplicating they don't seem to check `job_manager_user_can_post_job()`, which is what we hook into now.

tl;dr: It works, but if a better way eventually comes along then I'd like to change it.

**Comment by nosegraze** [comment]
Also just to add: because of the crappy hook I'm using, the error will be like `Invalid ID` instead of anything actually useful. That's not ideal, but it's what we've got.

**Comment by mihaijoldis** [comment]
@nosegraze as it is right now it seems to work.

**Comment by nosegraze** [comment]
@mihaijoldis I can also remove it from the front-end HTML too. Will push up an update shortly.

**Comment by mihaijoldis** [comment]
@nosegraze all good here !
